scripts/markdown-code-checker.py

```diff
@@ -7,10 +7,10 @@
 import subprocess
 
 
-def extract_inline_code(file_path, languages):
+def extract_inline_code(path, languages):
```

Comment:
This commit is not very nice, here is the commit message:

```
Changed var names, parameters and error handling
```

It would be much nicer to review if it was only changing the names and leaving the error handling for a separate commit. Also, in this commit you are removing the error handling (try except) only to reintroduce it / fix it in the next commit. Let's take a look at how to fix this together when you have time.